src/command/http-command.ts

```diff
@@ -106,8 +106,11 @@ export const httpCmd = (options: HttpOptions, resolverFn?: ResolverType): Reques
 			downloadLimit: 10_000,
 		},
 		agent: {
-			http: http.globalAgent,
-			https: new https.Agent({maxCachedSessions: 0}),
+			// Ensure Connection: closed header is used - https://nodejs.org/api/http.html#new-agentoptions
+			/* eslint-disable unicorn/prefer-number-properties */
```

Comment:
`eslint-disable` disables the rule until the end of the file, or until it stumbless accross `eslint-enable` comment.

Either use `eslint-disable-next-line` or add `eslint-enable` block below the `agent` object.

Reply:
@ayuhito this one was actually correct. I misread the second statement.

Both, your previous and the new solutions are fine by me. If you want to revert the comment - feel free to do so.

My bad.

Reply:
It's all good 👍 We can stick with this version.